src/transformers/searchservice/createDirectAnswer.ts

```diff
@@ -25,7 +25,14 @@ export function createDirectAnswer(data: any): FeaturedSnippetDirectAnswer | Fie
     };
   } else if (isFeaturedSnippetDirectAnswer) {
     const fieldType = commonDirectAnswerData.fieldType;
-    if (fieldType != BuiltInFieldType.MultiLineText && fieldType != BuiltInFieldType.RichText) {
+    const supportedFieldTypes: BuiltInFieldType[] = [
+      BuiltInFieldType.MultiLineText,
+      BuiltInFieldType.RichText,
+      BuiltInFieldType.RichText_v2,
+      BuiltInFieldType.Markdown,
+      BuiltInFieldType.Html
+    ];
+    if (!supportedFieldTypes.includes(fieldType)) {
```

Comment:
Is this error actually helpful for end-users and implementers? It seems like a type check on the API response which I'm not clear on the value of

Reply:
We thought it might be useful in case new field types get implemented for the API, but aren't added here. I don't think end-users should be seeing that often if this is kept up to date.

In the latest commit, we changed the method to return undefined if the type isn't supported (so it doesn't crash the application) and changed the error to a console warning.